File: comms/bulk-messaging/senders/search-senders/search-senders.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;
import com.twilio.sdk.resources.senders.requests.SendersSearchRequest;
import com.twilio.sdk.types.SenderCommunicationChannel;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN")
            .build();

        client.senders().search(
            SendersSearchRequest
                .builder()
                .address("+14153902337")
                .channel(SenderCommunicationChannel.SMS)
                .build()
        );
    }
}

File: comms/bulk-messaging/senders/search-senders/search-senders.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\Senders\Requests\SendersSearchRequest;
use Twilio\Comms\Types\SenderCommunicationChannel;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->senders->search(
    new SendersSearchRequest([
        'address' => '+14153902337',
        'channel' => SenderCommunicationChannel::Sms->value,
    ]),
);
